mesos-docker/run/run.sh

```diff
@@ -448,7 +482,7 @@ printMsg "Mesos cluster dashboard url http://$(docker_ip):5050"
 if [[ -n $INSTALL_HDFS ]]; then
   printMsg "Hdfs cluster started!"
   printMsg "Hdfs cluster dashboard url http://$(docker_ip):50070"
-  printMsg "Hdfs usrl http://$(docker_ip):8020"
+  printMsg "Hdfs usrl hdfs://$(docker_ip):8020"
```

Comment:
i print the hdfs url so it can be used elsewhere, for example if you run a hadoop command you need that... Previously i had it as http but it does not give you anything....just "It looks like you are making an HTTP request to a Hadoop IPC port. This is not the correct port for the web interface on this daemon."  Ideally you could have both...